examples/limitrange/valid-pod.json

```diff
@@ -1,20 +1,19 @@
 {
   "id": "valid-pod",
   "kind": "Pod",
-  "apiVersion":"v1beta2",
```

Comment:
I wanted to switch examples to v1beta3 but it doesn't look like it's enabled by default yet.